src/components/dropdown/dropdown.tsx

```diff
@@ -52,7 +54,8 @@ export type DropdownEqualityFn<T> = (item: T, value?: T) => boolean;
 export type DropdownValueComponent<T> = FC<{
   value?: T;
   disabled?: boolean;
-}>;
+}> &
+  TestIdProps;
```

Comment:
```suggestion
} & TestIdProps>;
```
Please check changes with prettier before applying